### PR TITLE
Add the kafka-XXX.knative-eventing.svc as hostname

### PR DESCRIPTION
**Pull request by Leo6Leo**
## Proposed Changes

- Add the kafka-broker-ingress.knative-eventing.svc as hostname

### CODE REVIEW

File: data-plane/config/broker-tls/broker-ingress-tls-certificate.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ spec:
 
   dnsNames:
     - kafka-broker-ingress.knative-eventing.svc.cluster.local
+    - kafka-broker-ingress.knative-eventing.svc
```
Comment (pierDipi) [inline data-plane/config/broker-tls/broker-ingress-tls-certificate.yaml:43]: Can we do this for other certificates? Including the one in eventing core?

Comment (Leo6Leo) [reply data-plane/config/broker-tls/broker-ingress-tls-certificate.yaml:43]: @pierDipi  Added for channel and sink. Also created the new issue in eventing core. https://github.com/knative/eventing/issues/7209

Is this what you are suggesting by saying "Including the one in eventing core" ?

**Review by Cali0707 — APPROVED** [review]
/lgtm
Thanks @Leo6Leo

**Comment by Cali0707** [comment]
@pierDipi are the failing tests here the loom tests you were mentioning on slack?

**Comment by matzew** [comment]
yes, loom is not expected to pass, yet (if I understood the message correctly)

**Comment by pierDipi** [comment]
> @pierDipi are the failing tests here the loom tests you were mentioning on slack?

yes

/unhold

**Review by matzew — APPROVED** [review]

**Pull request merged**